README.md

```diff
@@ -1,41 +1,82 @@
-$ yarn start
+git clone https://github.com/scaffold-eth/se2-docs.git
+cd se2-docs
+yarn install
+yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+## Contribution Guidelines
 
-### Build
+Thank you for your interest in contributing to improve the documentation!
 
-```
-$ yarn build
-```
+There are two types of contributions you can make:
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+- Fix errors or add new pages to the current documentation content.
+- Fix bugs or introduce new features to the documentation website (within the Docusaurus codebase).
 
-### Deployment
+## Content contributions
 
-Using SSH:
+The documentation content is written in Markdown format and is located in the `docs` folder. If you’re not familiar with Markdown, please check this [guide](https://guides.github.com/features/mastering-markdown/) or this [cheat sheet](https://www.markdownguide.org/cheat-sheet/) to get started.
 
-```
-$ USE_SSH=true yarn deploy
-```
+MDX files are also present, they combine Markdown with React. You can learn more about MDX files in [Docusaurus docs](https://docusaurus.io/docs/markdown-features/react).
 
-Not using SSH:
+### Existing content
 
+If you find anything that is confusing or can be improved in an existing document, you can click **"Edit this page" at the bottom of the page**, which will redirect to the GitHub edit form of that document. Make your changes and submit a pull request (PR).
+
+### New content
+
+When **adding a new page**, you need to fork the repository, create a new branch, and make all changes necessary in your repository. Once you are done with your changes, create a PR to Scaffold-ETH 2 Documentation repository.
+
+Add the new pages to the `docs` folder, placing them in the specific directory where you want the page to be shown.  
+**Siderbar** link will get **autogenerated** under your specific folder, in the **position** that you specify in your document header. Example:
+
+```markdown
+---
+sidebar_position: 2
+---
 ```
-$ GIT_USER=<Your GitHub username> yarn deploy
-```
 
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+If you want to **create a new folder** to the Document structure, you'll need to add the new folder and a markdown document with the same name of that folder, which will be displayed when the users click on the sidebar link.
+
+<!---
+TODO: Explain how to upload/link images to the docs. Currently there are none.
+-->
+
+<!---
+TODO: Explain _category_.json
+-->
+
+## Website contributions
+
+For contributions to the Website Code, fork the repository, create a new branch, and make all changes necessary in your repository. Once you are done with your changes, create a PR to Scaffold-ETH 2 Documentation repository.
+
```

Comment:
Should we mention about this : ["fork-and-pull" Git workflow](https://github.com/susam/gitpr) 

Like you can learn more about it here : ["fork-and-pull" Git workflow](https://github.com/susam/gitpr) ? 

we had this in SE-2 contribution.md -> https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md#pull-requests|

But feel free to omit it 🙌